uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -313,21 +315,23 @@ INPUTS:
   - $dbhr          : database handle reference
   - $db            : database object
   - $minc_location : location of the MINC files
+  - $uploadID      : ID of the upload containing the scan
 
 RETURNS: textual name of scan type from the C<mri_scan_type> table
 
 =cut
 
 sub identify_scan_db {
 
-    my  ($psc, $subjectref, $tarchiveInfoRef, $fileref, $dbhr, $db, $minc_location
-    ) = @_;
+    my  ($psc, $subjectref, $tarchiveInfoRef, $fileref, $dbhr, $db, $minc_location, $uploadID) = @_;
 
-    my $candid = ${subjectref}->{'CandID'};
-    my $pscid = ${subjectref}->{'PSCID'};
-    my $visit = ${subjectref}->{'visitLabel'};
+    my $candid       = ${subjectref}->{'CandID'};
+    my $pscid        = ${subjectref}->{'PSCID'};
+    my $visitLabel   = ${subjectref}->{'visitLabel'};
+    my $projectID    = ${subjectref}->{'ProjectID'};
```

Comment:
`ProjectID` and `SubprojectID` keys are not present in the `$subjectref` hash passed here so the only returned group is the default with value NULL for ProjectID, SubprojectID and visit label.